CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 3.6.0
+
+- Removes Billing permission from AndroidManifest since it's added by the BillingClient
+    https://github.com/RevenueCat/purchases-android/pull/211
+- Fixes Deferred downgrades. The Purchase object in the completion block of `purchaseProduct` and `purchasePackage` is now nullable when changing products.
```

Comment:
is this retrocompatible?

Reply:
partly, we deprecated the functions and made a new one that uses a new listener with a nullable purchase. The Kotlin conversion is not retrocompatible though so people accessing the purchase would get an error saying it's nullable now.

Reply:
in that case, should this be 4.0.0?

Reply:
If we are going to do a 4.0.0, we can remove the deprecated `makePurchase` call, right?

Reply:
YEP